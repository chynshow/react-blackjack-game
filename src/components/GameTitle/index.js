import React from 'react';

const GameTitle = () => (
  <h1 className='c-game-title'>
    <span className='c-game-title__decor'>_</span>Blackjack Game.
  </h1>
);

export default GameTitle;
