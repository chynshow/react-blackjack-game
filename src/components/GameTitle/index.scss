.c-game-title {
  font-size: 6rem;
  padding: 1rem 0;
  font-weight: 500;
  text-align: center;
  animation: fromLeft 1000ms linear both;

  @include respond(mobileL) {
    font-size: 4rem;
    padding: 1rem;
  }
}
