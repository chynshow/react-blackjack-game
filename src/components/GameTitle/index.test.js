import React from 'react';
import GameTitle from './index';
import { shallow } from 'enzyme';

it('render GameTitle component', () => {
  const c = shallow(<GameTitle />);
  expect(c).toMatchSnapshot();
});
