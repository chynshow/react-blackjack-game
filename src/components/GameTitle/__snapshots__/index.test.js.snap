// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render GameTitle component 1`] = `
<h1
  className="c-game-title"
>
  <span
    className="c-game-title__decor"
  >
    _
  </span>
  Blackjack Game.
</h1>
`;
